Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -294,3 +294,126 @@ angular.module("umbraco").controller("customBlockController", function ($scope,
+         "user_email":"janice@janiceindustries.com"
+      },
+      {
+         "contentTypeKey":"68075bd7-bd89-4614-9157-1979dba05768",
+         "udi":"umb://element/e51a247138594f6ca48f541d1bc1e09f",
+         "user_name":"John",
+         "user_email":"john@johnindustries.com"
+      }
+   ],
+   "settingsData":[
+      
+   ]
+}
+```
+
+We will be adding two people in `contentData` , whose `udi` values have to be referenced in the `layout` up above.
+`contentTypeKey` in this context is the Key value of the Document Type we are using in the Blocklist (`Person`), and the `udi` we will generate manually.
+
+To make things easier, we can make a basic model which we will later serialize into JSON:
+
+```csharp
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+    public class Blocklist //this class is to mock the correct JSON structure when the object is serialized
+    {
+        public BlockListUdi layout { get; set; }
+        public List<Dictionary<string, string>> contentData { get; set; }
+        public List<Dictionary<string, string>> settingsData { get; set; }
+    }
+    public class BlockListUdi //this is a subclass which corresponds to the "Umbraco.BlockList" section in JSON
+    {
+        [JsonProperty("Umbraco.BlockList")]  //we mock the Umbraco.BlockList name with JsonPropertyAttribute to match the requested JSON structure
+        public List<Dictionary<string, string>> contentUdi { get; set; }
+
+        public BlockListUdi(List<Dictionary<string, string>> items)
+        {
+            this.contentUdi = items;
+        }
+    }
+    public class Person //this is our Blocklist's allowed nested element. We make a model for it so we can create some dummy data
+    {
+        public string user_name { get; set; }
+        public string user_email { get; set; }
+        public Person(string user_name, string user_email)
+        {
+            this.user_name = user_name;
+            this.user_email = user_email;
+        }
+    }
+```
+
+After injecting [ContentService](../../../../../Reference/Management/Services/ContentService/) and [ContentTypeService](../../../../../Reference/Management/Services/ContentTypeService/), we can do the following:
+
+```csharp
+            var contentService = Services.ContentService; //if the class containing our code inherits SurfaceController, UmbracoApiController, or UmbracoAuthorizedApiController, we can get ContentService from Services namespace
+            IContentTypeService contentTypeService = Services.ContentTypeService;  //not to be confused with ContentService, this service will be useful for getting some Document Type IDs
+            GuidUdi contentUdi1 = new GuidUdi("element", System.Guid.NewGuid());  //we are creating two people to be added to Blocklist, which means we need two new Guids
+            GuidUdi contentUdi2 = new GuidUdi("element", System.Guid.NewGuid());  //Since these will be BLocklist items the Guids need to mention the "element" keyword
```

Comment:
GUID vs UDI - in the text you only mention udi, and now you're calling them guids. Perhaps consider clarifying the difference in the text.

```suggestion
            GuidUdi contentUdi2 = new GuidUdi("element", System.Guid.NewGuid());  //Since these will be BLock List objects the Guids need to mention the "element" keyword
```